CHANGES.rst

```diff
@@ -1,3 +1,8 @@
+v18.7.0
```

Comment:
This sounds like a breaking change. So maybe let's make it v19?
```suggestion
v19.0.0
```

Reply:
When I started dropping support for Python 2.7, I initially indicated a breaking change. Later, I noticed that thanks to python_requires, it's not really a breaking change as the packaging ecosystem maintains compatibility. By the same logic, adding a new dependency is technically a breaking change, but we don't treat it as one since the tooling handles it. My inclination is to stick with major change unless it's expected to require user intervention to account for the change.